### ISSUE TITLE
Rule proposal: html comment content newline

### ISSUE THREAD

**Issue by ota-meshi** [vuejs/eslint-plugin-vue#592]
**Please describe what the rule should do:**

Enforce newline before and after HTML comment content.

**What category of rule is this? (place an "X" next to just one item)**

[x] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```html
<!-- multiline
comment -->
```

```html
<!--

Too many newlines

-->
```

**Why should this rule be included in ESLint (instead of a plugin)?**

I think it should be included for the beauty of the Vue source code.

**Comment by ota-meshi** [comment]
I think it would be nice if there are rules that can keep the next style guide.

https://www.w3schools.com/html/html5_syntax.asp


### HTML Comments
Short comments should be written on one line, like this:

```html
<!-- This is a comment -->
```

Comments that spans more than one line, should be written like this:

```html
<!-- 
  This is a long comment example. This is a long comment example.
  This is a long comment example. This is a long comment example.
-->
```
